conf/modules.config

```diff
@@ -846,13 +846,22 @@ process{
         ]
     }
 
-    withName: 'GETPILEUPSUMMARIES.*' {
+     withName: 'GETPILEUPSUMMARIES.*' {
         ext.prefix       = { meta.num_intervals <= 1 ? meta.id : "${meta.id}_${intervals.simpleName}" }
         publishDir        = [
             mode: params.publish_dir_mode,
-            path: { "${params.outdir}/variant_calling/${meta.id}/" },
+            path: { "${params.outdir}/variant_calling/" },
+            pattern: "*.table",
+            saveAs: { meta.num_intervals > 1 ? null : "${meta.id}/mutect2/${it}" }
+        ]
+    }
+
+    withName: 'GETPILEUPSUMMARIES_.*' {
```

Comment:
why is there this selector twice?

Reply:
once with `_` (for `NORMAL` , `TUMOR` in paired mutect2 usage ) and the one without for all GETPILEUPSUMMARIES. Just added that to fix something :D

Reply:
hopefully I didn't break more stuff in the process 🐘 🍽️

Reply:
still same selector, the `saveAs` is overwritten

Reply:
yes but that is the point 🤔 for paired it should publish into ` "${meta.tumor_id}_vs_${meta.normal_id}/mutect2/${it}"` and for single it should publish into `"${meta.id}/mutect2/${it}"`

Reply:
:facepalm: 
Neverming, I got it, I saw the `...PILEUPSUMMARIES_.*` twice and on top of that did not pay enough attention to the `_` part...